lib/active_merchant/billing/gateways/checkout_v2.rb

```diff
@@ -163,6 +165,36 @@ def add_invoice(post, money, options)
         post[:metadata][:udf5] = application_id || 'ActiveMerchant'
       end
 
+      def add_recipient_data(post, options)
+        return unless recipient = options[:recipient]
+
+        post[:recipient] = {}
+        post[:recipient][:dob] = recipient[:dob] if recipient[:dob]
+        post[:recipient][:account_number] = recipient[:account_number] if recipient[:account_number]
+        post[:recipient][:zip] = recipient[:zip] if recipient[:zip]
```

Comment:
The [checkout docs](https://api-reference.checkout.com/#operation/requestAPaymentOrPayout!path=0/recipient/zip&t=request) say this is deprecated in favor of the `zip` nested under `address` so I think it's ok to remove this

Reply:
nice good catch!